node/impl/eth/transaction.go

```diff
@@ -412,7 +412,8 @@ func (e *ethTransaction) EthGetTransactionReceiptLimited(ctx context.Context, tx
 	if msgLookup == nil {
 		// This is the best we can do. In theory, we could have just not indexed this
 		// transaction, but there's no way to check that here.
-		return nil, nil
+		return nil, xerrors.Errorf("transaction receipt is too old (limit: %d epochs)", limit)
```

Comment:
This is a good addition, but you need to be careful of the case where `limit == api.LookbackNoLimit`, so do a check in here for `limit > api.LookbackNoLimit` like you do below and return `nil, nil` if the caller didn't actually pass a limit - because this is the case of just directly calling `EthGetTransactionReceipt()` directly above this function, and why the existing fevm_test.go integration test is failing in CI.

When you have it sorted, run `go test ./itests/fevm_test.go`, it'll take a little while but it'll run through all of the fevm tests and give you output if there's any errors.

Reply:
Alright. Will do